docs/documentation.rst

```diff
@@ -0,0 +1,19 @@
+:noindex:
+
+Documentation
+=================
+
+Target format and Airmass
+^^^^^^^^^^^^^^^^^^^^^^^^^
+.. automodule:: rise_set.astrometry
```

Comment:
The content for the "Target format and Airmass" and "Visibility API and Example Usage" sections aren't showing up in readthedocs... I think maybe these have to be `.. automodule:: astrometry` and `.. automodule:: visibility`

Reply:
Actually, I don't think this works